<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<author>Suraj Yadav</author>
	<class>EMST2DWindow</class>
	<widget class="QMainWindow" name="EMST2DWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>800</width>
				<height>600</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>CGAL EMST2DWindow</string>
		</property>
		<property name="windowIcon">
			<iconset resource="C:/ExtLibs/CGAL-4.9/demo/resources/CGAL.qrc">
				<normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
		</property>
		<widget class="QWidget" name="centralwidget">
			<layout class="QGridLayout">
				<item row="0" column="0">
					<widget class="QGraphicsView" name="graphicsView">
						<property name="enabled">
							<bool>true</bool>
						</property>
						<property name="focusPolicy">
							<enum>Qt::StrongFocus</enum>
						</property>
						<property name="verticalScrollBarPolicy">
							<enum>Qt::ScrollBarAlwaysOn</enum>
						</property>
						<property name="horizontalScrollBarPolicy">
							<enum>Qt::ScrollBarAlwaysOn</enum>
						</property>
						<property name="transformationAnchor">
							<enum>QGraphicsView::NoAnchor</enum>
						</property>
					</widget>
				</item>
			</layout>
		</widget>
		<widget class="QStatusBar" name="statusbar" />
		<widget class="QToolBar" name="fileToolBar">
			<property name="windowTitle">
				<string>File Tools</string>
			</property>
			<attribute name="toolBarArea">
				<enum>TopToolBarArea</enum>
			</attribute>
			<attribute name="toolBarBreak">
				<bool>false</bool>
			</attribute>
			<addaction name="actionClear" />
			<addaction name="actionLoadPoints" />
			<addaction name="actionSavePoints" />
			<addaction name="actionShowPoints" />
      <addaction name="actionShowMST" />
			<addaction name="actionShowDelaunay" />
			<addaction name="actionShowVoronoi" />
			<addaction name="actionRecenter" />
		</widget>
		<widget class="QMenuBar" name="menubar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>800</width>
					<height>26</height>
				</rect>
			</property>
			<widget class="QMenu" name="menuFile">
				<property name="title">
					<string>&amp;File</string>
				</property>
				<addaction name="separator" />
				<addaction name="actionClear" />
				<addaction name="actionLoadPoints" />
				<addaction name="actionSavePoints" />
				<addaction name="actionExport_SVG" />
				<addaction name="separator" />
				<addaction name="actionQuit" />
			</widget>
			<widget class="QMenu" name="menuTools">
				<property name="title">
					<string>&amp;Tools</string>
				</property>
				<addaction name="actionShowPoints" />
        <addaction name="actionShowMST" />
				<addaction name="actionShowDelaunay" />
				<addaction name="actionShowVoronoi" />
				<addaction name="separator" />
				<addaction name="actionRecenter" />
			</widget>
			<addaction name="menuFile" />
			<addaction name="menuTools" />
		</widget>
		<action name="actionAbout">
			<property name="text">
				<string>&amp;About</string>
			</property>
		</action>
		<action name="actionAboutCGAL">
			<property name="text">
				<string>About &amp;CGAL</string>
			</property>
		</action>
		<action name="actionExport_SVG">
			<property name="text">
				<string>&amp;Export SVG...</string>
			</property>
		</action>
		<action name="actionQuit">
			<property name="text">
				<string>&amp;Quit</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+Q</string>
			</property>
		</action>
		<action name="actionClear">
			<property name="icon">
				<iconset resource="C:/ExtLibs/CGAL-4.9/demo/icons/File.qrc">
					<normaloff>:/cgal/fileToolbar/fileNew.png</normaloff>:/cgal/fileToolbar/fileNew.png</iconset>
			</property>
			<property name="text">
				<string>&amp;Clear</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+C</string>
			</property>
		</action>
		<action name="actionShowPoints">
			<property name="checkable">
				<bool>true</bool>
			</property>
			<property name="checked">
				<bool>false</bool>
			</property>
			<property name="icon">
				<iconset>
					<normaloff>C:/ExtLibs/CGAL-4.9/demo/icons/Points.png</normaloff>C:/ExtLibs/CGAL-4.9/demo/icons/Points.png</iconset>
			</property>
			<property name="text">
				<string>Show &amp;Points</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+P</string>
			</property>
		</action>
    <action name="actionShowMST">
			<property name="checkable">
				<bool>true</bool>
			</property>
			<property name="checked">
				<bool>false</bool>
			</property>
			<property name="icon">
				<iconset>
					<normaloff>C:/ExtLibs/CGAL-4.9/demo/icons/Minimum_Spanning_Tree.png</normaloff>C:/ExtLibs/CGAL-4.9/demo/icons/Minimum_Spanning_Tree.png</iconset>
			</property>
			<property name="text">
				<string>Show &amp;Minimum Spanning Tree</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+M</string>
			</property>
		</action>
		<action name="actionShowVoronoi">
			<property name="checkable">
				<bool>true</bool>
			</property>
			<property name="checked">
				<bool>false</bool>
			</property>
			<property name="icon">
				<iconset>
					<normaloff>C:/ExtLibs/CGAL-4.9/demo/icons/Voronoi_diagram_2.png</normaloff>C:/ExtLibs/CGAL-4.9/demo/icons/Voronoi_diagram_2.png</iconset>
			</property>
			<property name="text">
				<string>Show &amp;Voronoi Diagram</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+V</string>
			</property>
		</action>
		<action name="actionShowDelaunay">
			<property name="checkable">
				<bool>true</bool>
			</property>
			<property name="icon">
				<iconset>
					<normaloff>C:/ExtLibs/CGAL-4.9/demo/icons/Delaunay_triangulation_2.png</normaloff>C:/ExtLibs/CGAL-4.9/demo/icons/Delaunay_triangulation_2.png</iconset>
			</property>
			<property name="text">
				<string>Show &amp;Delaunay Triangulation</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+D</string>
			</property>
		</action>
		<action name="actionLoadPoints">
			<property name="icon">
				<iconset resource="C:/ExtLibs/CGAL-4.9/demo/icons/File.qrc">
					<normaloff>:/cgal/fileToolbar/fileOpen.png</normaloff>:/cgal/fileToolbar/fileOpen.png</iconset>
			</property>
			<property name="text">
				<string>&amp;Load Points...</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+O</string>
			</property>
		</action>
		<action name="actionSavePoints">
			<property name="icon">
				<iconset resource="C:/ExtLibs/CGAL-4.9/demo/icons/File.qrc">
					<normaloff>:/cgal/fileToolbar/fileSave.png</normaloff>:/cgal/fileToolbar/fileSave.png</iconset>
			</property>
			<property name="text">
				<string>&amp;Save Points...</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+S</string>
			</property>
		</action>
		<action name="actionRecenter">
			<property name="icon">
				<iconset resource="C:/ExtLibs/CGAL-4.9/demo/icons/Input.qrc">
					<normaloff>:/cgal/Input/zoom-best-fit</normaloff>:/cgal/Input/zoom-best-fit</iconset>
			</property>
			<property name="text">
				<string>Re&amp;center the viewport</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+R</string>
			</property>
		</action>
	</widget>
	<resources>
		<include location="C:/ExtLibs/CGAL-4.9/demo/Triangulation_2/Delaunay_triangulation_2.qrc" />
		<include location="C:/ExtLibs/CGAL-4.9/demo/icons/File.qrc" />
		<include location="C:/ExtLibs/CGAL-4.9/demo/resources/CGAL.qrc" />
		<include location="C:/ExtLibs/CGAL-4.9/demo/icons/Triangulation_2.qrc" />
		<include location="C:/ExtLibs/CGAL-4.9/demo/icons/Input.qrc" />
	</resources>
	<connections/>
</ui>